libfreefare/freefare.3

```diff
@@ -117,6 +120,10 @@ can be gathered using the
 and
 .Fn freefare_get_tag_friendly_name
 functions.
+.Pp
+Use
+.Fn freefare_version
+to obtain a library version string
```

Comment:
:100: for thinking about the man page!

For consistency, can you reword this to something like `The freefare_version function returns the version of library.`?